packages/modular-scripts/src/esbuild-scripts/plugins/wsReload.ts

```diff
@@ -20,8 +21,14 @@ export default function createPlugin(
               name,
               building,
               result: {
-                errors: await Promise.all(result.errors.map(formatError)),
-                warnings: await Promise.all(result.warnings.map(formatError)),
+                errors: await Promise.all(
+                  result.errors.map((error) => formatError(error, baseDir)),
```

Comment:
Maybe since formatError is passed around so much we just make it an instance method of the `DevServer` so can just re-use it DRY-ly?

Reply:
Well, it's not passed around, it's `paths` that's passed around, and `formatError` is always imported. We can use a factory (or memoize) to associate an instance of it with a path and pass _that_ around, but I think it'd be messier (we also use it in build, so we'd need two instances). Now that we're using `paths` in our plugins, I think it looks clean.